include/ozo/connection_pool.h

```diff
@@ -9,24 +9,34 @@ namespace ozo {
 struct connection_pool_config {
     std::size_t capacity = 10;
     std::size_t queue_capacity = 128;
+    time_traits::duration idle_timeout = std::chrono::seconds(60);
+};
+
+struct connection_pool_timeouts {
```

Comment:
Could you please provide Doxygen compatible documentation about these fields.

Reply:
Done